lib/layout.js

```diff
@@ -139,12 +295,17 @@ export default class PodiumLayout {
         });
 
         // Join metric streams
+        // @ts-ignore
         this.httpProxy.metrics.pipe(this.metrics);
```

Comment:
I think we can get around these TS errors by adding `@types/readable-stream` as a devDependency.